cmd/panacead/cmd/jwt.go

```diff
@@ -0,0 +1,93 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/libs/cli"
+)
+
+func JwtCmd(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "issue-jwt [key-name] [expiration]",
+		Short: "Issue a JWT(Json Web Token) from account",
+		Long: `
+This command issue a JWT(Json Web Token) from account in panacea. 
+The key to be used for issuing a JWT should be stored in the localStore.
+If not stored, please add the key first.
+The [expiration] should use the format such as "1h", "30m", "2h45m" etc. 
+Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+		`,
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			fromAddress := clientCtx.GetFromAddress()
```

Comment:
When I try to use `--from` flag, `unknown flag` error occurs.
BTW, both `key-name`(arg[0]) and `--from` flag are required? IMO, we can use only `key-name` and get issuer address from it.

Reply:
Yes, we don't need to `--from` flag.

Reply:
Or how about using a `--from` flag? Because when issuing a JWT, I think that the issuer should be address of key. But I couldn't find a get address from [key-name]. So how about using a `--from` flag?

Reply:
It's up to you.

FYI, you can get address from private key as below
```go
btcecPrivKey, _ := btcec.PrivKeyFromBytes(btcec.S256(), privKeyBz)

privKey := secp256k1.PrivKey{
	Key: btcecPrivKey.Serialize(),
}

addr := sdk.AccAddress(priv.PubKey().Address()).String()
```

Reply:
Hmm.. Yes I think the converting is not safe. So I'll use `--from` flag!

Reply:
It seems if the private key is needed, we have only one option to use `key-name` argument and `UnsafeExportPrivKeyHex`.

Reply:
I didn't know the `GetFromName()`. Looks good!